test/e2e/kubelet_stats.go

```diff
@@ -159,17 +160,30 @@ func HighLatencyKubeletOperations(c *client.Client, threshold time.Duration, nod
 	return badMetrics, nil
 }
 
-// Retrieve metrics from the kubelet server of the given node.
-func getKubeletMetrics(c *client.Client, node string) (string, error) {
-	metric, err := c.Get().
+// Performs a get on a node proxy endpoing given the nodename and rest client.
+func nodeProxyRequest(c *client.Client, node, endpoint string) client.Result {
```

Comment:
nit: node doesn't seem like the right variable name here. nodeName?  I don't feel terribly strongly about this one.